kansha/card_addons/checklist/view.py

```diff
@@ -69,10 +74,9 @@ def render_Checklists(self, h, comp, model):
         with h.div(class_='checklists', id=id_):
             for index, clist in enumerate(self.checklists):
                 if can_edit:
-                    clist().init_input()
                     h << clist.on_answer(
                         lambda v, index=index: self.delete_checklist(index)
-                    ).render(h.AsyncRenderer())
+                    )
```

Comment:
When you click on a card, its checklists are reorderable.
If you add an item in one of them, it is not anymore, because the DOM has changed and the javascript to create the sortable has not been called again (and it must not as it creates the sortable for all the checklists).